google/cloud/credentials.h

```diff
@@ -51,6 +85,9 @@ class CredentialsVisitor;
  * @see https://cloud.google.com/docs/authentication for more information on
  *     authentication in GCP.
  *
+ * @see https://cloud.google.com/docs/iam for more information on the IAM
```

Comment:
404. I think you want https://cloud.google.com/iam/docs

Reply:
Fixed